<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="fr_FR" sourcelanguage="en_US">
<context>
    <name>DataController</name>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="80"/>
        <location filename="../src/lib/datacontroller.cpp" line="104"/>
        <source>Error</source>
        <translation>Erreur</translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="81"/>
        <source>The server reported an error.</source>
        <translation>Le serveur a reporté une erreur</translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="93"/>
        <source>If the problem persists, please contact the developer.</source>
        <translation>Si le problème persiste, veuillez contacter le développeur.</translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="100"/>
        <source>Network Error</source>
        <translation>Erreur Réseau</translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="101"/>
        <source>A network error occured</source>
        <translation>Une erreur réseau s&apos;est produite</translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="105"/>
        <source>An unknown error occured, we don&apos;t known more</source>
        <translation>Un erreur s&apos;est produite, nous n&apos;en savons pas plus</translation>
    </message>
</context>
</TS>
